test/passing/tests/ite-kw_first_no_indicate.ml.ref

```diff
@@ -159,3 +161,12 @@ let _ =
   else
     fun xxxxxxxxxxxxxxxxx yyyyyyyyyyyyyyyyyyyy zzzzzzzzzzz ->
     xxxxxxxxx yyyyyyyyyy zzzzzzzzzzzz
+
+let _ =
+  if (* foo *)
+     foo
+  then 0
+  else if (* bar *)
+          bar
```

Comment:
This one could be improved.

Reply:
This seems to be the intended behavior.

Reply:
After having a look at the diff, I'm not sure it looks better on "real" code, we can discard this commit.